demo-app/settings.gradle.kts

```diff
@@ -12,8 +14,8 @@ dependencyResolutionManagement {
         google()
     }
     versionCatalogs {
-        create("libs") {
```

Comment:
This block was only needed to reuse the same toml file from the root project. Since that's no longer the case, I think we can just remove the whole `versionCatalogs` block from this file, as the new catalog file `demo-app/gradle/libs.versions.toml` should be used automatically within the `demo-app` gradle files as `libs`.

Reply:
Ok, I'll remove this and then change `buildLibs` back to `libs` primarily because it's less code....but I wonder if we'll get ourselves confused not knowing which catalog we're looking at sometimes. I know I struggled with that once or twice while getting this going.

Reply:
Ah, gotcha. I didn't think it could be confusing. That's fair enough, I'm in favor of changing it if that makes it easier to read 👍